hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneHAClusterImpl.java

```diff
@@ -966,4 +970,31 @@ public StorageContainerManager getStorageContainerManager() {
     return getStorageContainerManagers().get(0);
   }
 
+  private int getFreePort() {
```

Comment:
It would be good to reuse `SCMTestUtils.getReusableAddress()` here rather than creating a new version in this class. It returns an InetSocketAddress, but we can easily get the port from it.